gate-core/src/main/java/com/netflix/spinnaker/gate/retrofit/UpstreamBadRequest.java

```diff
@@ -36,6 +38,14 @@ private UpstreamBadRequest(RetrofitError cause) {
     error = cause.getBody();
   }
 
+  private UpstreamBadRequest(SpinnakerHttpException cause) {
+    super(cause.getMessage(), cause);
+    this.setRetryable(false);
```

Comment:
Wouldn't
```suggestion
    this.setRetryable(cause.getRetryable());
```
make more sense here?  Are there tests that could tell the difference?

Reply:
Agreed. Made the changes. PermissionServiceTest.lookupServiceAccountFailuresAreMarkedRetryable() test has various scenarios on this.